operations/customers.md

```diff
@@ -156,6 +156,7 @@ Returns all customers filtered by identifiers, emails, names and other filters.
 | `UpdatedUtc` | string | required | Last update date and time of the customer in UTC timezone in ISO 8601 format. |
 | `ItalianDestinationCode` | string | optional | Value of Italian destination code. |
 | `ItalianFiscalCode` | string | optional | Value of Italian fiscal code. |
+| `CompanyId` | string | optional | Unique identifier of the company. |
```

Comment:
I'd add link to company object and expanded the description little more.
```suggestion
| `CompanyId` | string | optional | Unique identifier of [Company](enterprises.md#company) the customer is associated with. |
```

Reply:
Ok, will fix

Reply:
Also you could extend the sample response. All optional parameters doesn't need to be there, but this one is useful.

Reply:
Updated